tests/benchmarks/test_futures.py

```diff
@@ -5,7 +5,8 @@
 
 def test_single_future(small_client):
     """How quickly can we run a simple computation?"""
-    small_client.submit(inc, 1).result()
+    for i in range(400):
```

Comment:
A comment as to why we're looping here would be useful for future readers. We might also want to update the test name as it doesn't seem fully right anymore with these changes.

Reply:
Also, you mentioned 400 iterations causes this test to take ~41s. Can we have fewer iterations  while still making the wall time long enough to meaningfully measure differences? For example, is 100 iterations enough?

Reply:
Good question -- maybe from some of the plots in https://github.com/coiled/coiled-runtime/issues/420?